college_flow_app/lib/features/list_subjects/widgets/searchbar.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:college_flow_app/config/design_system/data/spacing/spacing.dart';
+import 'package:college_flow_app/shared/widgets/flow_icon_button.dart';
+import 'package:college_flow_app/shared/widgets/gap.dart';
+import 'package:flutter/material.dart';
+
+import '../../../config/design_system/data/colors/colors.dart';
+import '../../../shared/widgets/flow_icon.dart';
+
+class SearchBar extends StatelessWidget {
+  const SearchBar({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      mainAxisAlignment: MainAxisAlignment.center,
+      mainAxisSize: MainAxisSize.max,
+      children: [
+        FlowIconButton(
+          icon: const FlowIcon.search(
+            color: colorSecondary,
+            size: spacingXS,
+          ),
+          onTap: () {},
+        ),
```

Comment:
why use FlowIConButton and not FLowIcon? If you aren't using the onTap param, change to FlowIcon

Reply:
the idea is to do the search when the user tap the icon